ec2utils/ec2uploadimg/python-ec2uploadimg.spec

```diff
@@ -18,7 +18,7 @@
 
 %define upstream_name ec2uploadimg
 Name:           python-ec2uploadimg
-Version:        2.0.0
+Version:        2.1.0
```

Comment:
We use semantic versioning, you changed the interface to EC2ImageUploader thus this will result in 3.0.0